_data/blogs.yml

```diff
@@ -97,3 +97,8 @@
   title: MY EXPERIENCE OF GOOGLE CODE – IN
   img: aks.jpg
   author_name: Ashish Kumar Singh
+
+- url: https://morethanworld64569539.wordpress.com/2017/12/12/how-i-fixed-an-issue-by-20-steps/
+  title: How I fixed an issue? – By 20 steps
+  img: supun3.png
```

Comment:
![image](https://user-images.githubusercontent.com/24358501/33939952-1c67643e-e033-11e7-8530-8b44df6087c9.png)
both are same image @STEdirisuriya

Reply:
What is the problem? Please see above of Blogs section there is three same photos. Two picture are from one person. And other same picture from another person. May be twins.

Reply:
See it first.